### ISSUE TITLE
[BUG] Null check operator used on a null value

### ISSUE THREAD

**Issue by mkranus** [jhomlala/betterplayer#663]
**Describe the bug**
Non-nullable value was null at runtime

**To Reproduce**

No exact steps, error occurs randomly due to unknown reason

**Example code**
```
BetterPlayerControlsState.skipBack() - Line 41
BetterPlayerControlsState.skipForward() - Line 52
```

**Expectation**
The function should just exit if the value is null, instead of throwing an error.

**Flutter doctor**
Flutter (Channel stable, 2.2.3, on macOS 11.5.2 20G95 darwin-x64, locale en-US)
    • Flutter version 2.2.3 at /Users/_/flutter
    • Framework revision f4abaa0735 (8 weeks ago), 2021-07-01 12:46:11 -0700
    • Engine revision 241c87ad80
    • Dart version 2.13.4

**Better Player version**
- Version: 0.0.73

**Smartphone (please complete the following information):**
 - Device: iPhone 11 Pro
 - OS: 14.6.0 (18F72)

**Comment by jhomlala** [comment]
Fixed in 0.0.74.

**Comment by masoudk2000** [comment]
@jhomlala
I have the same error, i got this report in Sentry.

reported stack trace :

_CastError: Null check operator used on a null value
  File "framework.dart", line 1108, in State.setState
  File "better_player_material_progress_bar.dart", line 38, in new _VideoProgressBarState.<fn>
  File "change_notifier.dart", line 308, in ChangeNotifier.notifyListeners
  File "change_notifier.dart", line 412, in ValueNotifier.value=
  File "video_player.dart", line 421, in VideoPlayerController.dispose

I used better player 0.77
